src/js/scenes/OldMain/Root.js

```diff
@@ -17,26 +17,26 @@ We are no longer using "react-router-native" in this way
 
 
 /****************************** ROUTE-COMPONENTS ******************************/
-//import About from "./routes/More/About";
+//import About from "./routes/SignIn/About";
```

Comment:
Hi Steve, it looks like there was a search-and-replace from "More" to "SignIn" that wasn't what you had planned.

Reply:
Must have been the Webstorm "refactor"  ... good catch.